kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/yaml/consistency/YamlDataConsistencyCheckResult.java

```diff
@@ -32,6 +32,10 @@ public final class YamlDataConsistencyCheckResult implements YamlConfiguration {
     
     private YamlDataConsistencyContentCheckResult contentCheckResult;
     
+    private String checkIgnoredType;
```

Comment:
Could `checkIgnoredType` be `ignoredType`? Then it's consistent with `ignored` field